lms/envs/common.py

```diff
@@ -3564,3 +3564,6 @@ def _make_locale_paths(settings):
 # CDN LINK
 
 CDN_LINK = "https://static.philanthropyu.org/"
+
+# PHILU Configuration Parameters.
+PHILU_DEFAULT_TERM_SEARCH_LENGTH = 2
```

Comment:
1. Let's please not define it in common.py because this is not something generic It is specific to org search. common.py is a core file. We can define a constant either in the same function or max in constants file in onboarding app.
2. Let's please not append PHILU with any name, reason is already discussed alot. and let's make it specific instead of generic search term something like `ORG_SEARCH_TERM_LENGTH`, `ORG_SEARCH_MIN_QUERY_LENGTH`